src/pages/søknad/index.module.less

```diff
@@ -17,7 +17,7 @@
 }
 
 .stegContainer {
-    width: 100%;
+    width: 105%;
```

Comment:
Dette virker litt skummelt 😅  Kunne vi ha tatt en titt på om det gikk an å løse det på en annen måte?

Reply:
Ja, dette ble egentlig den siste løsningene etter jeg tok det opp med Agnethe, for nå. 

Til info, dem 3 input feltene, med validering, får dem til å ikke lenger bli alignet, og med blir klistret på hverandre.